insights/client/collection_rules.py

```diff
@@ -139,22 +138,15 @@ def validate_gpg_sig(self, path, sig=None):
         Validate the collection rules
         """
         logger.debug("Verifying GPG signature of Insights configuration")
+
         if sig is None:
             sig = path + ".asc"
-        command = ("/usr/bin/gpg --no-default-keyring "
-                   "--keyring " + constants.pub_gpg_path +
-                   " --verify " + sig + " " + path)
-        if not six.PY3:
-            command = command.encode('utf-8', 'ignore')
-        args = shlex.split(command)
-        logger.debug("Executing: %s", args)
-        proc = Popen(
-            args, shell=False, stdout=PIPE, stderr=STDOUT, close_fds=True)
-        stdout, stderr = proc.communicate()
-        logger.debug("STDOUT: %s", stdout)
-        logger.debug("STDERR: %s", stderr)
-        logger.debug("Status: %s", proc.returncode)
-        if proc.returncode:
+
+        result = crypto.verify_gpg_signed_file(
+            file=path, signature=sig,
+            keys=[constants.pub_gpg_path],
```

Comment:
The "infrastructure" to load multiple keys already exists in the crypto module. This will make adding the support for extra GPG keys much easier. I don't see what the difference between a single path and a list of paths is.